web-client/src/presenter/actions/navigateToCaseDetailWithDocketNumberAction.js

```diff
@@ -0,0 +1,22 @@
+import { state } from 'cerebral';
+
+/**
+ * changes the route to view the case-detail of the caseId of props.caseId
```

Comment:
Should this be docketNumber instead for the references to caseId in the comments?

Reply:
Yes! Good catch!

Reply:
Updated!